src/libostree/ostree-repo-pull.c

```diff
@@ -3853,6 +3853,8 @@ ostree_repo_pull_with_options (OstreeRepo             *self,
 
       while (g_variant_iter_loop (collection_refs_iter, "(&s&s&s)", &collection_id, &ref_name, &checksum))
         {
+          if (!ostree_validate_rev (ref_name, error))
```

Comment:
We also need this in the next `else if` for `refs_to_fetch`, right?

Reply:
There's already one there, no?  Line 3879?

Reply:
Whoops! Indeed.